spec/domain/mailing_lists/bulk_mail/retriever_spec.rb

```diff
@@ -337,6 +337,30 @@
       expect(message.subject).to eq("Anlass hinzugef�gt")
       expect(message.state).to eq('pending')
     end
+
+    it 'does process mail with empty subject and enqueues job for mail delivery' do
+      imap_mail = build_imap_mail(42, nil)
+      allow(imap_connector).to receive(:fetch_mail_by_uid).with(42, :inbox).and_return(imap_mail)
+
+      expect(imap_mail).to receive(:original_to).and_return('leaders@localhost:3000')
+      expect(imap_connector).to receive(:delete_by_uid).with(42, :inbox)
+      expect(imap_mail_validator).to receive(:sender_allowed?).and_return(true)
+
+      expect do
+        retriever.perform
+      end.to change { Message::BulkMail.count }.by(1)
+                                               .and change { MailLog.count }.by(1)
+                                                                            .and change { Delayed::Job.where('handler like "%Messages::DispatchJob%"').count }.by(1)
+                                                                                                                                                              .and change { Delayed::Job.where('handler like "%MailingLists::BulkMail::SenderRejectedMessageJob%"').count }.by(0)
```

Comment:
Can we indent this a little? Or, if that would be ridiculous, align the "change" so that it is readable. I think auto-indenting is a bit broken here. If Rubocop complains, please tell me.

This version is unreadable.